src/block/index.js

```diff
@@ -50,6 +52,11 @@ module.exports = (bridgeState, db, nodeConfig = {}) => async (
   bridgeState.currentState = state;
   bridgeState.blockHeight = chainInfo.height;
 
-  await bridgeState.saveNodeState();
```

Comment:
not saving proposal data every block. Instead saving every 32nd block (see `startNewPeriod`)